components/button.json

```diff
@@ -180,7 +202,8 @@
     },
     "hyperlink": {
       "#normal": {
-        "text": "@theme-text-accent-normal"
+        "text": "@theme-text-accent-normal",
+        "border": "@theme-background-ghost"
       },
```

Comment:
add a transparent border for normal theme. desktop needs normal state in validator, just have focused will cause the build to fail

Reply:
removed border for now until we decide how this should be handled